app/assets/javascripts/modules/track-email-alert-signup-choices.js

```diff
@@ -0,0 +1,43 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (global, GOVUK) {
+  'use strict'
+
+  var $ = global.jQuery
```

Comment:
There's a movement, that is gathering steam, to not write new code that requires jQuery based on the [gds-way](https://gds-way.cloudapps.digital/manuals/programming-languages/js.html#jquery). 

Since these changes introduce jQuery to this project it'd probably save some pain if this was in vanilla JS. I know this creates a bit of a pain as it's a port though it looks like jQuery is only used for DOM traversal and setting an event handler.

Reply:
/cc @alex-ju it'd be useful to get your perspective on what approach we should be taking for porting jQuery code into projects which currently don't have jQuery. I do feel a bit mean about making the suggestion of changing but does seem to be reflective of the govuk-frontenders stance as per yesterdays meeting?

Reply:
@laurentqro we recently made a decision at the community level that we must stop introducing more jQuery code in our efforts to move away from jQuery and align with the GDS way as Kevin mentioned above. Since this script is relatively straight-forward, I will have to ask for it to be written in pure JavaScript and use the [reference module pattern](https://github.com/alphagov/govuk_publishing_components/blob/master/docs/javascript-modules.md#module-structure). [Here](https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/javascripts/govuk_publishing_components/components/contextual-guidance.js) is a decent example of how the pattern is used in practice and I am happy to answer any questions related to this refactor.